libs/uriparser/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2015-2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uriparser
+PKG_VERSION:=0.9.7
+PKG_RELEASE:=1
+
+PKG_SOURCE:=uriparser-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/uriparser/uriparser/releases/download/uriparser-$(PKG_VERSION)/
+PKG_HASH:=11553b2abd2b5728a6c88e35ab08e807d0a0f23c44920df937778ce8cc4d40ff
+
+PKG_MAINTAINER:=Weijia Song <songweijia@gmail.com>,Sebastian Pipping <sebastian@pipping.org>
```

Comment:
Who are these maintainers? Are there aware that they will be maintainers?

Reply:
It would be if he is active, tries to contribute and take cares of the feedback, which does not happen. So, the maintainer should be someone, who is interested in it and will update it in the future. We can not add new packages without maintainer.